### PR TITLE
Rename props with attrs

### LINKED ISSUE #77: VElement: props vs attributes
The HTML element attributes in a `VElement` are stored in a field called `props`.

Why `props` and not `attrs` or `attributes`? If there's no particular reason behind this I'd suggest a rename to `attributes` to [match the DOM](https://developer.mozilla.org/en-US/docs/Web/API/Element/attributes).

(I suspect it was simply copied over from the JS tutorial on which the virtual dom implementation was based on.)

### PR DESCRIPTION
**Pull request by tock203**
While reading source code of virtual_dom_rs, I was confused by VElement's `props`. [the term "properties" and "attributes" has slightly different meanings in DOM](https://stackoverflow.com/a/6004028). VElement's one named `props` but what it contains is "attributes" rather than "properties" I think.

Fixes #77

### CODE REVIEW

File: crates/virtual-node/src/lib.rs [hunk 1]
```diff
@@ -72,7 +72,7 @@ pub struct VElement {
     /// The HTML tag, such as "div"
     pub tag: String,
     /// HTML props such as id, class, style, etc
```
Comment (chinedufn) [inline crates/virtual-node/src/lib.rs]: Mind just changing `props` to `attributes` in this comment?

File: crates/virtual-node/src/lib.rs [hunk 2]
```diff
@@ -394,7 +394,7 @@ impl fmt::Debug for VElement {
         write!(
             f,
             "Element(<{}>, props: {:?}, children: {:?})",
```
Comment (chinedufn) [inline crates/virtual-node/src/lib.rs]: Mind changing `props` to `attrs` here?

File: crates/virtual-node/src/lib.rs [hunk 3]
```diff
@@ -410,7 +410,7 @@ impl fmt::Display for VElement {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         write!(f, "<{}", self.tag).unwrap();
 
-        for (prop, value) in self.props.iter() {
+        for (prop, value) in self.attrs.iter() {
```
Comment (chinedufn) [inline crates/virtual-node/src/lib.rs]: Same here `for (attr`

File: crates/virtual-node/src/virtual_node_test_utils.rs [hunk 4]
```diff
@@ -126,7 +126,7 @@ mod tests {
         let mut props = HashMap::new();
```
Comment (chinedufn) [inline crates/virtual-node/src/virtual_node_test_utils.rs]: `attrs` here

File: crates/html-macro-test/src/lib.rs [hunk 5]
```diff
@@ -33,7 +33,7 @@ fn one_prop() {
     let mut props = HashMap::new();
```
Comment (chinedufn) [inline crates/html-macro-test/src/lib.rs]: `attrs` here

**Comment by tock203** [comment]
oh I totally forgot about those... fixed it.

**Comment by chinedufn** [comment]
Thanks so much !!!

**Pull request merged**